src/components/EmailNotifications/VerifyEmailAddress.js

```diff
@@ -0,0 +1,251 @@
+                      onBlur={handleEmailAddressBlur}
+                    />
+                    {emailInvalid && (
+                      <p
+                        css={`
+                          color: ${theme.negative};
+                          ${textStyle('body4')};
+                          float: left;
+                          margin-top: ${0.5 * GU}px;
+                        `}
+                      >
+                        Please enter a valid email address
+                      </p>
+                    )}
+                  </Field>
+                </div>
+                <LegalTermsAndPolicy
+                  termsAccepted={termsAccepted}
+                  setTermsAccepted={setTermsAccepted}
+                  theme={theme}
+                />
+                <div
+                  css={`
+                    display: flex;
+                    justify-content: space-between;
+                    flex-direction: ${compactMode ? 'column' : 'row'};
+                    width: 100%;
+                    margin-bottom: ${1.5 * GU}px;
+                    margin-top: ${3 * GU}px;
+                  `}
+                >
+                  <ActionButtons
+                    compactMode={compactMode}
+                    onClick={onDeleteEmail}
+                    width={buttonWidth}
+                  >
+                    Delete email
+                  </ActionButtons>
+                  <ActionButtons
+                    compactMode={compactMode}
+                    mode="strong"
+                    disabled={emailInvalid || !termsAccepted}
+                    onClick={handleSubscribe}
+                    width={buttonWidth}
+                  >
+                    Send verification email
+                  </ActionButtons>
+                </div>
+              </>
+            )
+          }
+          return (
+            <Button
+              css={`
+                margin-top: ${3 * GU}px;
+              `}
+              mode="strong"
+              onClick={handleReSendEmail}
+            >
+              Resend verification email
+            </Button>
+          )
+        })()}
+      </div>
+    </div>
+  )
+})
+
+function LegalTermsAndPolicy({ termsAccepted, setTermsAccepted, theme }) {
+  return (
+    <div
+      css={`
+        margin-top: ${3 * GU}px;
+      `}
+    >
+      <div
+        css={`
+          display: flex;
+        `}
+      >
+        <Checkbox
+          checked={termsAccepted}
+          onChange={checked => setTermsAccepted(checked)}
+        />
+
+        <span
+          css={`
+            ${textStyle('body2')};
+            color: ${theme.surfaceContentSecondary};
+            margin-left: ${1.5 * GU}px;
+            text-align: left;
+          `}
+        >
+          {`By continuing with your email, you agree `}
+          <Link href="#">legal terms </Link> and{' '}
+          <Link href="#"> email collection policy.</Link>
```

Comment:
Do we have the URLs for these? Also I think the sentence should say “you agree with our legal terms and […]”.

Reply:
ahh good catch, will add a TODO to add the links once we have them